native-image-config/README.md

```diff
@@ -0,0 +1,46 @@
+# How to generate GraalVM native image configuration
+
+## Step 1: Generate native image configuration files
+
+First, generate the native image configuration files by running the tests annotated with
+`GenerateNativeImageTrace` with [`native-image-agent`](https://www.graalvm.org/latest/reference-manual/native-image/metadata/AutomaticMetadataCollection/).
+
+```
+$ ../gradlew nativeImageConfig
```

Comment:
When running from my local env. I found the following logs:
```
Warning: Error processing trace entry map(size=5, {(tracer,reflect),(function,getEnclosingMethod),(caller_class,org.awaitility.core.AssertionCondition),(result,false),(args,[])}): java.lang.StringIndexOutOfBoundsException: begin 0, end -1, length 5
        at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4606)
        at java.base/java.lang.String.substring(String.java:2709)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.trace.ReflectionProcessor.addFullyQualifiedDeclaredMethod(ReflectionProcessor.java:293)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.trace.ReflectionProcessor.processEntry(ReflectionProcessor.java:247)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.trace.TraceProcessor.processEntry(TraceProcessor.java:87)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.trace.TraceProcessor.processTrace(TraceProcessor.java:63)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.trace.TraceProcessor.process(TraceProcessor.java:58)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.command.ConfigurationGenerateCommand.generate(ConfigurationGenerateCommand.java:293)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.command.ConfigurationGenerateCommand.apply(ConfigurationGenerateCommand.java:60)
        at org.graalvm.nativeimage.configure/com.oracle.svm.configure.ConfigurationTool.main(ConfigurationTool.java:84)
```

Just want to make sure this is normal/expected

Reply:
It was reported to https://github.com/oracle/graal/issues/6980